index.html

```diff
@@ -111,6 +111,11 @@ <h1 id="title">
         There has been no change in dependencies on other workings groups
         during the development of this specification.
       </p>
+      <div class="correction">
+        This specification includes <a href=
+        "https://www.w3.org/Consortium/Process/#candidate-correction">Candidate
+        corrections</a>, which are marked in the document.
+      </div>
```

Comment:
My reading of the process document is that these would be classified as "candidate additions", not corrections. 

```suggestion
      <p>
        This specification includes <a href=
        "https://www.w3.org/2020/Process-20200915/#candidate-addition">Candidate
        additions</a>, which are marked in the document.
      </p>
```